plugins/Referrers/API.php

```diff
@@ -337,6 +323,36 @@ public function getSocials($idSite, $period, $date, $segment = false, $expanded
         $this->setSocialIdSubtables($dataTable);
         $this->removeSubtableMetadata($dataTable);
 
+        if ($flat) {
+            $urlsTable = Archive::createDataTableFromArchive(Archiver::WEBSITES_RECORD_NAME, $idSite, $period, $date, $segment, $expanded, $flat);
```

Comment:
Here we need a special handling to make it fast as `getSocials` does not really have a subtable...